Source/Plugins/Processing/ProcessingFilters/MinNeighbors.cpp

```diff
@@ -183,6 +182,11 @@ void MinNeighbors::dataCheck()
   }
 
   getDataContainerArray()->validateNumberOfTuples(this, dataArrayPaths);
+
+  // Throw a warning to inform the user that the neighbor list arrays could be deleted by this filter
+  QString ss = QObject::tr("If this filter changes the selected feature ids array, all NeighborList arrays will be deleted.");
```

Comment:
Change the warning to:
If this filter changes the selected Feature Ids Array located at (INSERT THE PATH TO THE SELECTED ARRY), all AttributeArrays of type NeighborList located in the AttributeMatrix (GIVE PATH TO THE PROPER ATTRIBUTEMATRIX) will be DELETED.